README.md

```diff
@@ -84,9 +85,14 @@ variables might be required for proper scaling and resolutions:
 | BALENA_HOST_CONFIG_hdmi_mode          | 87                 |
 | BALENA_HOST_CONFIG_rotate_screen      | 1                  |
 
+### D-Bus
+
+The D-Bus container exists to enable the optional display on your Pi-hole device for [PADD](https://github.com/pi-hole/PADD).
```

Comment:
We can probably remove this line since dbus is just a workaround to avoid display conflicts. As long as we indicate it's expected to stop that should be enough?

Either that or change it to something like this:

```
The D-Bus service is a workaround to prevent display buffer conflicts when running development images.
It is expected behaviour for this container to run once upon boot and then exit.
It will remain in `Exited` status during normal operation.
```

Reply:
Haha - I actually had the workaround in there at first but then took it out because I wasn't 100% sure of the purpose.  Always go with your first instincts, Roddie.